src/application.ts

```diff
@@ -107,7 +111,8 @@ export default class Application implements Owner {
     if (!mainTemplate) { throw new Error("Could not find main template."); }
 
     let mainLayout = templateFactory(mainTemplate).create(this.env);
```

Comment:
@robbiepitts basically the build pipeline compiles it and we just ship it as part of the bundle like any other templates. but for now if you couldn't figure if out, a hack would be to run a build on an app with that template, and then vendor the JSON output here. Obviously if you do that we will need to clean it up very soon, but it should work and will unblock you